x-pack/solutions/security/plugins/session_view/public/components/tty_player_controls/tty_player_controls_markers/styles.ts

```diff
@@ -36,7 +41,7 @@ export const useStyles = (progress: number) => {
       if (selected) {
         return euiVars.terminalOutputMarkerAccent;
       }
-      return euiVars.euiColorVis1;
+      return isAmsterdam(themeName) ? euiVars.euiColorVis1 : euiVars.euiColorVis2;
```

Comment:
check color palette matching table in https://docs.google.com/document/d/1IAKbasq1nDfqd2IU3KdP8cwD3uCCAwkIekKRq7zgyWg/edit?tab=t.0#heading=h.u0vaano75gy0